HARSHAL/Reactjs/react_app/src/components/assignment2.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from "react";
+
+export default class IncrementDecrement extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            count: 0,
+        };
+    }
+    changecountincrement = () => {
```

Comment:
This Ok method name, but try make method name like this:
easy to Understand.
```suggestion
    changeCountIncrement = () => {
```